ci/shared.sh

```diff
@@ -21,3 +24,14 @@ function retry {
 
   return ${exit_code}
 }
+
+function mkcargotemp {
+  local td=
+  mkdir -p "$ci_dir"/../target/tmp
```

Comment:
Might be worth having a `package_dir` or `project_dir` folder at this point.

Reply:
I'm not sure I follow, do you mean instead of doing a temporary folder keeping it in tree (i.e check it out instead of writing files on demand for tests)

Reply:
Nope, I mean we have:

```bash
ci_dir=$(dirname "${BASH_SOURCE[0]}")
ci_dir=$(realpath "${ci_dir}")
export PROJECT_HOME=$(dirname "${ci_dir}")
```

And then we can use it here as `${PROJECT_HOME}/target/tmp` and also in the other scripts.